src/bindings/io/uv/example/exampleio_uv.c

```diff
@@ -127,11 +127,7 @@ int main (void)
 	// Start the event loop
 	uv_run (loop, UV_RUN_DEFAULT);
 
-#ifdef HAVE_LIBUV1
 	uv_loop_close (loop);
-#elif HAVE_LIBUV0
-	uv_loop_delete (loop);
```

Comment:
Why remove support for older libs?

Reply:
When fixing static & full builds with the "io_uv" binding I ran into problems. But I could solve them and restore support for libuv 0.x.